api/HouseOutput.cs

```diff
@@ -11,15 +19,101 @@ namespace api
  */
 public class HouseOutput : IDeviceOutput
 {
-	public HouseOutput()
+	public HouseOutput(string house_info, string device_info)
 	{
+		_houseInfo = house_info;
+		_deviceInfo = device_info;
 	}
 
 	public bool write(Device dev)
 	{
-		//TODO: Implement writing to the house
+		string json = JsonConvert.SerializeObject(dev);
+		string houseID = dev.ID.HouseID.ToString();
+		string roomID = dev.ID.RoomID.ToString();
+		string deviceID = dev.ID.DeviceID.ToString();
+
+		var _ = writeHelper(json, houseID, roomID, deviceID);
```

Comment:
Really? I was running into issues with the variable being set to the task object, and requiring a `Wait()` call to get a value. But I haven't dug in enough to be certain - I've just been helping people with HTTP calls they've been working on. If it passes the unit test, then it's clearly working, and it's good to go. I'll check out and verify via unit test once I get a chance.